src/target/ext/base/single_register_access.rs

```diff
@@ -35,7 +35,7 @@ pub trait SingleRegisterAccess<Id>: Target {
         &mut self,
```

Comment:
meta: you'll also need to update the `read_register` docs to rephrase the following into something more correct:

```
/// Implementations should write the value of the register using target's
/// native byte order in the buffer `dst`
```